### PR TITLE
Added player statistic charts

### PR DESCRIPTION
**Pull request by Mailaender**
I combined the data from http://openra.ipdx.ru/graphs.html with a modern AJAX interface from http://javascriptrrd.sourceforge.net/docs/javascriptrrd_v1.1.1/src/examples/rrdJFlot.html

image

image

image

image

The options have been reduced and I chose some sane defaults to render readable graphs. The statistics are linked from http://www.openra.net/games/ and use the same color theme.

:bulb: **Pro-Tip**: You can hover over the graph ~~and also zoom in with the mouse~~.

### CODE REVIEW

**Comment by pchote** [comment]
Looks like a reasonable compromise to me, but there are still some polish issues:
* The padding on the tooltip doesn't match on all sides.
* "Waiting: 2 on 19:25" doesn't make sense - the tooltips should use "at" for the hourly plot.  I don't think that plot gives us a lot, so wouldn't mind just dropping it.
* The stroke on the two curves looks messy, you could try reducing the stroke width or removing it completely.
* The plot on the games page should use a tooltip, since you're now set on including it.

File: content/games.html [hunk 1]
```diff
@@ -104,14 +104,50 @@
...
+<h3>Server Browser</h3>
+<p><span id="activePlayers">0</span> <a href="/players">players</a> are online in <span id="activeServers">0</span> advertised servers.</p>
 <table>
   <thead><tr><th>Server</th><th>Status</th><th>Mod</th><th>Map</th></tr></thead>
   <tbody id="serverlist"></tbody>
 </table>
+<br/>
+<h3>Player Activity</h3>
+<p>View the numbers of gamers <font color="#0A0">playing</font> and <font color="#F80">waiting</font> over the last fortnight:</p>
+<a href="/players"><div id="players"><center><h4>Loading...</h4></center></div></a>
```
Comment (pchote) [inline content/games.html]: These links are missing the trailing slash, which makes them 404.

File: content/games.html [hunk 2]
```diff
@@ -104,14 +104,50 @@
...
+      use_checked_DSs: true, checked_DSs: ['playing', 'waiting'],
+      graph_width: "620px", graph_height: "200px"
+    };
+
+    plot = new rrdFlotAsync("players", rrdfile, null, graph_options, ds_graph_options, rrdflot_options);
+  })
+</script>
+
+<h3>Server Browser</h3>
+<p><span id="activePlayers">0</span> <a href="/players">players</a> are online in <span id="activeServers">0</span> advertised servers.</p>
```
Comment (pchote) [inline content/games.html]: There's two links below, so this one is probably overkill.

**Comment by Mailaender** [comment]
Updated.

**Comment by pchote** [comment]
You missed my last set of suggestions, so I fixed them (and a few of the earlier "hard" ones) myself: ed9f7d0379b7c4727bf5053f5ea0735355bb7540

**Comment by Mailaender** [comment]
Updated with your suggestions. Thanks.

Not quite sure about your choices of size plot thickness and color temperature. It is hard to see and navigate with tooltips now.

image

**Comment by obrakmann** [comment]
:+1: let's get this done

**Comment by pchote** [comment]
The `<font>` tag is long-deprecated, and is invalid html5.  This causes the playing and waiting labels to be rendered black in IE11 on windows.  Use a `<span>` with `class="gamePlaying"` or `gameWaiting` instead.

**Comment by Mailaender** [comment]
Updated.

**Comment by pchote** [comment]
The automated deployment appears to have failed, but there are no error messages explaining why.  I guess that travis is missing the uglifier package?

**Comment by Mailaender** [comment]
Deployment failed and we don't know why thanks to https://github.com/OpenRA/OpenRAWeb/pull/179.

**Comment by Mailaender** [comment]
The deployment key has been used so I think the error is in my script. I will try to use another way to still hide my token from the logs: https://github.com/OpenRA/OpenRAWeb/pull/183 should fix it.

**Comment by Mailaender** [comment]
I deployed this manually test my new scripts. https://github.com/OpenRA/OpenRAWeb/pull/189

**Pull request merged**